ws-client/src/client.rs

```diff
@@ -179,6 +181,7 @@ pub struct WsClientBuilder<'a> {
 impl<'a> Default for WsClientBuilder<'a> {
 	fn default() -> Self {
 		Self {
+			use_system_certificates: false,
```

Comment:
I won't mind changing `WsClient` to use the `native certificate store` by default, we use it in the `HttpClient` for example

Reply:
Ok :)

Would you also prefer i called `use_system_certificates` something like `use_native_certificate_store`?

Reply:
Personally, I would prefer to have an enum configure the certificate store, boolean for that seems it little bit hard to understand.

such as:

```
enum CertificateStore {
  Native,
  WebPki,
}

/// Set certificate store to use
///
/// Default: native certificate store
pub fn certificate_store(mut self, cert: CertificateStore) -> Self {
		self.certificate_store = cert;
		self
	}
```

But you could just remove the `use_system_certificates option and hardcode it to native then fine for me, `webpiki` seems more secure but I think it's too complicated to use for most users anyway.

Reply:
Do you think exposing [RootCertStore](https://docs.rs/rustls/0.19.1/rustls/struct.RootCertStore.html) would be to take it too far?

Then the user could do what ever they wanted. We could still have `rustls_native_certs::load_native_certs()` as default.

Reply:
let's ignore for it now, I think most users wants us to control that but perhaps add an issue for it.